pipelines/test/ldd-check.yaml

```diff
@@ -0,0 +1,59 @@
+name: ldd-check
+
+needs:
+  packages:
+    - busybox
+    - posix-libc-utils
+
+inputs:
+  files:
+    description: |
+      The files to run `ldd` on and check for missing deps.
+    required: true
+  verbose:
+    description: |
+      Should the full ldd output be shown
+    required: false
+    default: false
+
+pipeline:
+  - name: "run ldd on provided files"
+    runs: |
+      set +x
```

Comment:
.... did you mean to leave this here?

Reply:
Yes. It just makes noise. Makes output harder to read